test/TestProjects/MgmtSafeFlatten/Generated/Models/LayerOneBaseType.cs

```diff
@@ -28,5 +30,13 @@ internal LayerOneBaseType(LayerOneTypeName name)
 
         /// <summary> Gets or sets the name. </summary>
         internal LayerOneTypeName Name { get; set; }
+
+        /// <summary> Create an wrapped subclass instance of LayerOneBaseType by serializing the provided BinaryData. </summary>
+        /// <param name="data"> The BinaryData that will be used in serializing, it should be a valid json object. </param>
+        /// <returns> The wrapped subclass instance of LayerOneBaseType. </returns>
+        public static LayerOneBaseType FromBinaryData(BinaryData data)
```

Comment:
Lets keep this out of the first version.  I believe this was in relation to the round trip of unknown types?  For now we recognize that round trip of unknown types can result in an issue lets address that in a future change

Reply:
Removed